test/cmdlineTests/~bytecode_equivalence_multiple_sources/inputs.sol

```diff
@@ -0,0 +1,44 @@
+==== Source: A.sol ====
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.0;
```

Comment:
Also, we could make the test more concise by extracting preamble. You could define a helper function for adding preamble to all `*.sol` files in a directory and run it at the beginning of the script.

Reply:
Added.